src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int weight;
```

Comment:
It seems there is a typo in the variable name 'weight'. In the context of a rectangle, the correct term should be 'width'. Please rename it to reflect the intended meaning.